test/serdes_test.c

```diff
@@ -95,14 +95,6 @@ static int test_serialize_deserialize(const char *type, EVP_PKEY *pkey,
         && !TEST_ptr(EVP_PKEY_get0(pkey2)))
         goto end;
 
-    /*
-     * TODO(3.0) Remov this when PEM_write_bio_PrivateKey_traditional()
-     * handles provider side keys.
-     */
-    if (make_legacy
```

Comment:
Did you mean to remove the code also?

Reply:
Yes, it was a duplicate of the same lines above...  that probably happened when I rebased this after #12410 was merged